.github/workflows/ci.yml

```
@@ -0,0 +1,282 @@
+        
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v2
+      with:
+        version: ${{ env.PNPM_VERSION }}
+        
+    - name: Install dependencies
+      run: pnpm install --frozen-lockfile
+      
+    - name: Run security audit
+      run: pnpm audit --audit-level moderate
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)